src/providers/services/auth/auth.service.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment.prod';
```

Comment:
Use relative path not absolute. Have a look at every import if their is any change it.
And import environment from environment file not from environment.prod file as for production environment file automatically replaced by prod file